angular/src/app/viewer/visualization/network-display/network-display.component.ts

```diff
@@ -18,34 +16,34 @@ import { BackendService } from 'src/app/shared/services/backend.service';
   styleUrls: ['./network-display.component.scss'],
 })
 export class NetworkDisplayComponent implements AfterViewInit {
-  public selectedLayout: any = { value: 'concentric', id: 'Concentric' };
-  public network: Network;
-
-  @ViewChild('layoutPicker')
-  layoutPicker!: LayoutPickerComponent;
+  // @ViewChild('layoutPicker')
+  // layoutPicker!: LayoutPickerComponent;
+  // public layout: string = 'concentric';
```

Comment:
Might be okay to delete these, unless you're planning on uncommenting in a future refactoring or something.